### PR TITLE
feat(#10): added prompt to chose between yarn and npm

### PR DESCRIPTION
**Pull request by alemagio**
## Description

Added new prompt to allow the user to chose if he/she wants to use yarn or npm as package manager.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

[Here](https://github.com/lirantal/create-node-lib/issues/10)

## How Has This Been Tested?

I just tested the minor part that i believe need a test.
## Checklist:

- [ ] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I added a picture of a cute animal cause it's fun

### CODE REVIEW

**Comment by alemagio** [comment]
Still need to add tests but if you want to take a look feel free to comment @lirantal .

**Comment by alemagio** [comment]
Even if I'm actually not sure what to test in this scenario, it's all on sao.

File: template/package.json [hunk 1]
```diff
@@ -193,7 +193,7 @@
         "changelogFile": "CHANGELOG.md"
       }
     ],
-    "@semantic-release/npm",
+    "@semantic-release/<%= npmClient %>",
```
Comment (lirantal) [inline template/package.json:196]: this should still be npm :)
```suggestion
    "@semantic-release/npm",
```

File: template/package.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
     "node": ">=10.0.0"
   },
   "scripts": {
-    "lint": "eslint . && npm run lint:lockfile",
-    "lint:lockfile": "lockfile-lint --path package-lock.json --type npm --validate-https --allowed-hosts npm yarn",
+    "lint": "eslint . && <%= npmClient %> run lint:lockfile",
+    "lint:lockfile": "lockfile-lint --path package-lock.json --type <%= npmClient %> --validate-https --allowed-hosts npm yarn",
```
Comment (lirantal) [inline template/package.json:11]: so here there is a `package-lock.json` which is for npm, and we need to make it either that, or `yarn.lock` if the user chooses yarn so we'll need a conditional here.

**Comment by alemagio** [comment]
Hi, I made the changes you requested. Please consider I'm new to sao and so if you think there is a better way to achieve our goal just let me know.
I'm here to learn :sweat_smile:

**Comment by lirantal** [comment]
Looks great to me @alemagio, great job! 🥳👏

**Comment by alemagio** [comment]
> Looks great to me @alemagio, great job! 🥳👏

Still missing dog picture actually.
Next pr, promise.

**Pull request merged**